provider/k8s/process.go

```diff
@@ -573,6 +575,34 @@ func (p *Provider) podSpecFromRunOptions(app, service string, opts structs.Proce
 		s.Containers[0].Resources.Limits["memory"] = resource.MustParse(fmt.Sprintf("%dMi", *opts.MemoryLimit))
 	}
 
+	p.DockerUsername = common.CoalesceString(os.Getenv("DOCKER_HUB_USERNAME"),"")
+	p.DockerPassword = common.CoalesceString(os.Getenv("DOCKER_HUB_PASSWORD"),"")
```

Comment:
do not initialize the var here. you should initialize that in here: https://github.com/convox/convox/blob/master/provider/k8s/k8s.go#L94